deployment/overlays/memtierd/daemonset.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nri-memtierd
+  name: nri-memtierd
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: nri-memtierd
+  template:
+    metadata:
+      labels:
+        app: nri-memtierd
+    spec:
+      nodeSelector:
+        kubernetes.io/os: "linux"
+      hostPID: true
+      containers:
+        - name: nri-memtierd
+          command:
+            - nri-memtierd
+            - --idx
+            - "45"
+            - --config
+            - /etc/nri/memtierd/config.yaml
+            - --host-root
+            - /host
+            - -v
+          image: IMAGE_PLACEHOLDER
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 250m
+              memory: 100Mi
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - name: memtierd-config-vol
+            mountPath: /etc/nri/memtierd
+          - name: nri-sockets-vol
+            mountPath: /var/run/nri
+          - name: host-vol
+            mountPath: /host
```

Comment:
Is this correct path or should it be `/host/sys/kernel/...`?

Reply:
The thing is that `memtierd` launched by `nri-memtierd` does not currently support host root mounting. It seems that Luukas had found this workaround, which indeed gives writes access to the bitmap file directly in /proc inside the nri plugin container. That said, `nri-memtierd` currently uses at least `/host/tmp`.